CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -42,11 +43,15 @@ public IList<HarvestDeveloper> RetrieveDevelopers()
             ).ToList(); 
         }
 
-        public IList<TimeSheet> RetrieveTimeSheets()
+        public IList<TimeSheet> RetrieveTimeSheets(DateTimeOffset dateFrom, DateTimeOffset dateTo)
         {
             _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _token);
- 
-            var apiResponse = GetApiResponse("/api/v2/time_entries").Result;
+            
+            var dateFromFmt = dateFrom.ToString("yyyy-MM-dd");
```

Comment:
Could create an extension method from DateTimeOffset called something like ToHarvestApiString() or something better than that.

`
public static string ToHarvestApiString(this DateTimeOffset date){
    return date.ToString("yyyy-MM-dd");
}
`

`
var dateFromFmt = dateFrom.ToHarvestApiString();
var dateToFmt = dateTo.ToHarvestApiString();
`

Reply:
Nice Idea Lad. Added as a private method in Harvest Gateway as it wouldn't be used anywhere else in the project. 
https://github.com/chubberlisk/crypto_tech/pull/23/commits/dff0c84e5ce8032e8720f02637eac19f5042a54a